From ec406d667a5591af359da7bbae29d15b4b1589bf Mon Sep 17 00:00:00 2001
From: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Date: Thu, 24 Mar 2022 17:37:19 +0200
Subject: [PATCH 055/471] ASoC: dt-bindings: mchp,i2s-mcc: add
 'sound-name-prefix' property

Add 'sound-name-prefix' property to differentiante between interfaces in
DPCM (ASRC) use-cases. Property is optional.

Signed-off-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
---
 .../devicetree/bindings/sound/mchp,i2s-mcc.yaml          | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Documentation/devicetree/bindings/sound/mchp,i2s-mcc.yaml b/Documentation/devicetree/bindings/sound/mchp,i2s-mcc.yaml
index 0481315cb5f2..c9ec85ce6fdb 100644
--- a/Documentation/devicetree/bindings/sound/mchp,i2s-mcc.yaml
+++ b/Documentation/devicetree/bindings/sound/mchp,i2s-mcc.yaml
@@ -66,6 +66,15 @@ properties:
     enum: [0, 1, 2, 3]
     default: 0
 
+  sound-name-prefix:
+    pattern: "^I2SMCC[0-9]$"
+    $ref: /schemas/types.yaml#/definitions/string
+    description:
+      used as prefix for sink/source names of the component. Must be a
+      unique string among multiple instances of the same component.
+      The name can be "I2SMCC0" or "I2SMCC1" ... "I2SMCCx", where x depends
+      on the maximum available instances on a Microchip SoC.
+
 if:
   properties:
     compatible:
-- 
2.34.1

